src/arviz_plots/plots/ppctstat.py

```diff
@@ -0,0 +1,214 @@
+    aes_map=None,
+    plot_kwargs=None,
+    stats_kwargs=None,
+    pc_kwargs=None,
+):
+    """
+    Plot Bayesian t-stat for observed data and posterior/prior predictive.
+
+    Parameters
+    ----------
+    dt : DataTree
+        Input data
+    group : str,
+        Group to be plotted. Defaults to "posterior_predictive".
+        It could also be "prior_predictive".
+    t_stat : str, float, default "median"
+        Test statistics to compute from the observations and predictive distributions.
+        Allowed strings are “mean”, “median” or “std”. Alternative a quantile can be passed
+        as a float (or str) in the interval (0, 1).
+    var_names : str or list of str, optional
+        One or more variables to be plotted.
+        Prefix the variables by ~ when you want to exclude them from the plot.
+    filter_vars : {None, “like”, “regex”}, default=None
+        If None, interpret var_names as the real variables names.
+        If “like”, interpret var_names as substrings of the real variables names.
+        If “regex”, interpret var_names as regular expressions on the real variables names.
+    sample_dims : str or sequence of hashable, optional
+        Dimensions to reduce unless mapped to an aesthetic.
+        Defaults to ``rcParams["data.sample_dims"]``
+    kind : {"kde", "hist", "dot", "ecdf"}, optional
+        How to represent the marginal density.
+        Defaults to ``rcParams["plot.density_kind"]``
+    plot_collection : PlotCollection, optional
+    coords : dict, optional
+    backend : {"matplotlib", "bokeh", "plotly"}, optional
+    labeller : labeller, optional
+    data_pairs : dict, optional
+        Dictionary of keys prior/posterior predictive data and values observed data variable names.
+        If None, it will assume that the observed data and the predictive data have
+        the same variable name.
+    aes_map : mapping of {str : sequence of str}, optional
+        Mapping of artists to aesthetics that should use their mapping in `plot_collection`
+        when plotted. Valid keys are the same as for `plot_kwargs`.
+    plot_kwargs : mapping of {str : mapping or False}, optional
+        Valid keys are:
+
+        * One of "kde", "ecdf", "dot" or "hist", matching the `kind` argument.
+
+          * "kde" -> passed to :func:`~arviz_plots.visuals.line_xy`
+          * "ecdf" -> passed to :func:`~arviz_plots.visuals.ecdf_line`
+          * "hist" -> passed to :func: `~arviz_plots.visuals.hist`
+
+        * credible_interval -> passed to :func:`~arviz_plots.visuals.line_x`
+        * point_estimate -> passed to :func:`~arviz_plots.visuals.scatter_x`
+        * point_estimate_text -> passed to :func:`~arviz_plots.visuals.point_estimate_text`
+        * title -> passed to :func:`~arviz_plots.visuals.labelled_title`
+        * rug -> passed to :func:`~arviz_plots.visuals.scatter_x`. Defaults to False.
+        * remove_axis -> not passed anywhere, can only be ``False`` to skip calling this function
+
```

Comment:
credible_interval, credible_interval, point_estimate_text and rug defaults to False..
        Pass an (empty) mapping to plot them.

Reply:
We are defaulting to False, but the user can still "turn on" these graphical elements. Thus, we should explain how to do it, like in my suggested change. Additionally, we should pass the top-level arguments `point_estimate`, `ci_kind` and `ci_prob` to `plot_dist`, because they affect these graphical elements and clafirfy in the docstring that they have no effect unless the elements are active.

Reply:
Have added params to plot_ppc_tstat, with default as False